### ISSUE TITLE
Phantomjs failure on win10. "Cliver::Dependency::Not Found"

### ISSUE THREAD

**Issue by shadow1129** [jhu-ep-coursera/fullstack-course1-module3-blackbox-testing#2]
I followed the steps in the videos to do the blackbox-testing for "my_first_app". However, failures were encountered (shown as below).

I have installed Phantomjs and add the path to environment variables. As a result, I am able to check the version of it.

But when running "rspec", it showed an error that

"Cliver::Dependency::NotFound:

Could not find an executable at given path 'c:/Program Files/phantomjs-2.1.1-windows/bin/phantomjs.EXE'.If this path was not specified explicitly, it is probably a bug in [Cliver](https://github.com/yaauie/cliver/issues)."

![2016-12-08_1-42-39](https://cloud.githubusercontent.com/assets/14102363/21032459/8d1be9e6-bd77-11e6-900a-815982ca1e1a.png)
![2016-12-08_1-54-15](https://cloud.githubusercontent.com/assets/14102363/21032463/9078a5e8-bd77-11e6-9a9d-5464cb450599.png)

I was wondering whether it's an issue of the version of phantomjs I use (2.1.1) or something else. If it is, how can I backtrack to a previous version?

Thanks.

**Comment by shadow1129** [comment]
Thanks to Mark Sun in Coursera, the issue is resolved by moving the phantom folder out of "Program Files" folder and changing "PATH" accordingly.

**Comment by kalmanh** [comment]
@SpartaSixZero is the man indeed! :) 

Thank you Mark Sun!

**Comment by ejavaguy** [comment]
Yes - Although "phantomjs --version" will work with it installed in
"Program Files", phantomjs (at least 2.1.1) must be installed in a
directory without spaces in it for Capybara to find it at test time. I
moved mine to c:/apps/ and all worked fine.

**Comment by SpartaSixZero** [comment]
Haha, no problem! This is actually a known issue in the past. I did a search in Coursera and found some very similar questions. Gotta give credit to past mentors like Aina. I saw her answer worked so I just used that one.

**Comment by kalmanh** [comment]
Mark, Pls add it to the FAQs

**Comment by SpartaSixZero** [comment]
I was just thinking about doing that! I will add that in.

**Comment by kalmanh** [comment]
Thanx!
